heartbeat/azure-events.in

```diff
@@ -221,7 +222,7 @@ class clusterHelper:
 
 		nodes = []
 		nodeList = clusterHelper._exec("crm_node", "--list")
-		for n in nodeList.split("\n"):
+		for n in nodeList.decode().split("\n"):
```

Comment:
We should try to not use some trivial names as `n`. It could be named `node` for example